src/material/stepper/stepper.md

```diff
@@ -142,59 +128,19 @@ set via `<mat-icon>` elements. If you want to provide a different set of icons,
 by placing a `matStepperIcon` for each of the icons that you want to override. The `index`,
 `active`, and `optional` values of the individual steps are available through template variables:
 
-```html
-<mat-vertical-stepper>
-  <ng-template matStepperIcon="edit">
-    <mat-icon>insert_drive_file</mat-icon>
-  </ng-template>
-
-  <ng-template matStepperIcon="done">
-    <mat-icon>done_all</mat-icon>
-  </ng-template>
-
-  <!-- Custom icon with a context variable. -->
-  <ng-template matStepperIcon="number" let-index="index">
-    {{index + 10}}
-  </ng-template>
-
-  <!-- Stepper steps go here -->
-</mat-vertical-stepper>
-```
+<!-- example({"example":"stepper-states",
+              "file":"stepper-states-example.html", 
+              "region":"override-icons"}) -->
 
 Note that you aren't limited to using the `mat-icon` component when providing custom icons.
 
 #### Step States
 You can set the state of a step to whatever you want. The given state by default maps to an icon.
 However, it can be overridden the same way as mentioned above.
 
-```html
-<mat-horizontal-stepper>
-  <mat-step label="Step 1" state="phone">
-    <p>Put down your phones.</p>
-    <div>
-      <button mat-button matStepperNext>Next</button>
-    </div>
-  </mat-step>
-  <mat-step label="Step 2" state="chat">
-    <p>Socialize with each other.</p>
-    <div>
-      <button mat-button matStepperPrevious>Back</button>
-      <button mat-button matStepperNext>Next</button>
-    </div>
-  </mat-step>
-  <mat-step label="Step 3">
-    <p>You're welcome.</p>
-  </mat-step>
-
-  <!-- Icon overrides. -->
-  <ng-template matStepperIcon="phone">
-    <mat-icon>call_end</mat-icon>
-  </ng-template>
-  <ng-template matStepperIcon="chat">
-    <mat-icon>forum</mat-icon>
-  </ng-template>
-</mat-horizontal-stepper>
-```
+<!-- example({"example":"stepper-states",
```

Comment:
We usually have a space between after the colon in object literals, e.g. `"example": "steper-states"`.